rest/src/test/kotlin/request/MessageRequests.kt

```diff
@@ -1,42 +0,0 @@
-package dev.kord.rest.request
-
-import dev.kord.rest.NamedFile
-import dev.kord.rest.json.response.GatewayResponse
-import dev.kord.rest.route.Route
-import io.ktor.util.*
-import kotlinx.coroutines.runBlocking
-import org.junit.jupiter.api.MethodOrderer
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.TestInstance
-import org.junit.jupiter.api.TestMethodOrder
-import java.io.InputStream
-
-@TestMethodOrder(MethodOrderer.MethodName::class)
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
-class MessageRequests {
```

Comment:
Could (and should) we bring this test back but with `assert(!readChannel.isClosedForWrite)` before and `assert(readChannel.isClosedForRead)` after executing a mock request (since the whole point is that the channel is consumed lazily)?